cpp/CMakeLists.txt

```diff
@@ -85,6 +88,9 @@ endif()
 
 ##############################################################################
 # - compiler options ---------------------------------------------------------
+set(CMAKE_C_COMPILER_LAUNCHER ccache)
+set(CMAKE_CXX_COMPILER_LAUNCHER ccache)
+set(CMAKE_CUDA_COMPILER_LAUNCHER ccache)
```

Comment:
Shouldn't ccache be in the yml for this?

Reply:
It got added to the conda yml [here](https://github.com/rapidsai/rapids-triton/pull/9/files#diff-5a7cdc513dfd4ea76a879c3bb55b92c9093ff19661ca03564224e315e68e22c7R6). I assume that's what you meant?

Reply:
yeah, hadn’t spotted it